<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="note_content">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="location">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="starttime">
       <property name="text">
        <string>Start time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End time</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="start_time_content">
       <property name="text">
        <string>Start time content</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="end_time_content">
       <property name="text">
        <string>End time content</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="note">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>99</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Rec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DirBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>160</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="From_content">
       <property name="text">
        <string>From_content</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="To_content">
       <property name="text">
        <string>To_content</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
